resources/function_help/json/rotate

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "rotate",
+  "type": "function",
+  "description": "Returns a rotated version of a geometry. Calculations are in the Spatial Reference System of this geometry.",
+  "arguments": [ {"arg":"geom","description":"a geometry"},
+                 {"arg":"rotation","description":"clockwise rotation in degrees"},
```

Comment:
Generally we try to follow to approach used by the correponsing postgis function. In this case that would be st_rotate, which uses "radians counter-clockwise about the origin point".

Reply:
-1
Personally I'm in favour of using degrees instead of radians. As a GIS guy I'm expect directions and angles to be in degrees and clockwise with 0 = north. I also think most users do, and their data is likely to contain values in degrees.

Also, QgsGeometry().rotate() expects values in degrees and I would prefer being consistent  within QGIS than following the approach used in other software (postgis). And also the angle_at_vertex() expression function returns values in degrees

Reply:
It's a recurring debate. One day, I hope that we can add the option to choose the unit (degree, rad, grad) and the orientation (clockwise, counter, geographic).

When I develop, I am always embarrassed to use the geographical north as point 0.
But indeed, GIS users (especially QGIS?), have this habit.

So +0.5 with @raymondnijssen

Reply:
No strong preference, I can live with both.
Just pretty please let's keep expressions portable and not add an option. I remember a spreadsheet application that chose to go for translated function names depending on the system language ...

Reply:
```suggestion
                 {"arg":"rotation","description":"clockwise rotation from north in degrees"},
```
? though not the exact wording in the angle_at_vertex description.

Reply:
@DelazJ The "from north" would not mean anything here, since the angle defines a rotation (= relative) and not a direction (= absolute).

Reply:
Ok, fair enough!